tests/integration/reports/single_table/_properties/test_column_shapes.py

```diff
@@ -66,3 +67,24 @@ def test_get_score_warnings(self):
         assert error_messages[0] == expected_message_1
         assert error_messages[1] == expected_message_2
         assert score == 0.826
+
+    def test__generate_details_end_to_end(self):
+        # Setup
+        real_data, synthetic_data, metadata = load_demo('single_table')
+        column_shape_property = ColumnShapes()
+        quality_report = QualityReport()
+        quality_report.generate(real_data, synthetic_data, metadata)
+        details_quality_report = quality_report.get_details(property_name='Column Shapes')
+        details_quality_report = details_quality_report.rename(columns={'Quality Score': 'Score'})
+
+        # Run
+        column_shape_property.get_score(real_data, synthetic_data, metadata)
+
+        # Assert
+        details_column_shape = column_shape_property._details
```

Comment:
Idk if we're testing much by just testing the output matches the `_details`. We should test the values match the expected calculations. Maybe on a smaller dataset or just for the head

Reply:
I'm testing that the `details` table is the same if it's generated by the `Quality report` or by the `property`.  Shouldn't this ensure that the values match the expected calculations? Because it should fail if the scores of the `Score` column are different. I can rename the test.

Reply:
This test should be removed after we changed the Quality Report to use the property (I'm doing this task)

Reply:
Yes, that's true because the `QualityReport` will use the property, so the details table will be the same